test/integration/framework/src/siftool/cosmos.py

```diff
@@ -32,8 +42,8 @@ def balance_neg(bal: Balance) -> Balance:
     return {k: -v for k, v in bal.items()}
 
 
-def balance_sub(bal1: Balance, bal2: Balance) -> Balance:
-    return balance_add(bal1, balance_neg(bal2))
+def balance_sub(bal1: Balance, *bal2: Balance) -> Balance:
+    return balance_add(bal1, *[balance_neg(b) for b in bal2])
```

Comment:
balance_add just take one parameter

Reply:
No, the `*` in Python stands for argument splicing, both `balance_add` and `balance_sub` support a list of arguments. With `*` you can write instead of `balance_sub(a, [b, c, d])` just `balance_sub(a, b, c, d)`.
The semantics is `a - b - c - d`.